spec.md

```diff
@@ -534,9 +534,16 @@ Credentials are a set of information used by an application or a user to utilize
 
 #### Log Drain ####
 
-There are a class of service offerings that provide aggregation, indexing, and analysis of log data. To utilize these services an application that generates logs needs information for the location to which it will stream logs. If a broker represents one of these services, it MAY return a `syslog_drain_url` in the response for a request to create a service binding, to which logs MUST be streamed.
+There are a class of service offerings that provide aggregation, indexing, and
+analysis of log data. To utilize these services an application that generates
+logs needs information for the location to which it will stream logs. A
+create binding response from one of these services MUST include a
+`syslog_drain_url`. Applications MUST use this URL when sending logs to the
+service.
```

Comment:
> Applications MUST use this URL when sending logs to the service.

In the case of CF, it is the platform that sends logs to the service, not directly the application, see http://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html The wording might therefore be confusing to CF community.